databases/week2/hw_week2.sql

```diff
@@ -0,0 +1,69 @@
+--Part 1: Working with tasks
+--1. Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (id, title, description, created, updated, due_date, status_id, user_id) VALUES (36, 'Learn HYF Week2 lesson', 'ERD, Normalization, etc.', '2022-08-22 03:41:53', '2022-08-25 09:19:56', '2022-10-11 10:13:42', 2, NULL);
```

Comment:
In general we never insert the primary key (`id` in this case) manually, we just let the system do it for us.
When creating the tables the primary key field is usually created with the AUTO_INCREMENT keyword, which means that the primary key will be automatically inserted by the system. This will avoid stuff like inserting a primary key  value that already exists and will throw an error :)